protocol/avdl/paperprovision.avdl

```diff
@@ -0,0 +1,15 @@
+
+@namespace("keybase.1")
+protocol paperprovision {
+
+  import idl "common.avdl";
+
+  /**
+    Performs paper provision.
+    username is optional.
+    If the current device isn't provisioned, this function will
+    provision it.
+    */
+  void paperProvision(int sessionID, string username, string deviceName, string paperKey);
```

Comment:
I thought you could just put this in `login.avdl`, but since you went through all the plumbing for its own protocol, this is fine.